src/components/parallax-scroll.component.js

```diff
@@ -30,34 +24,37 @@ type State = {
   parallaxHeaderHeight: number,
 };
 
-const styles = StyleSheet.create({
-  background: {
-    position: 'absolute',
-    top: 0,
-    backgroundColor: colors.primaryDark,
-  },
-  stickySection: {
-    height: STICKY_HEADER_HEIGHT,
-    backgroundColor: colors.primaryDark,
-    alignItems: 'center',
-    justifyContent: 'flex-end',
-  },
-  stickySectionText: {
-    color: colors.white,
-    ...fonts.fontPrimaryBold,
-    fontSize: normalize(16),
-    margin: 10,
-  },
-  fixedSectionLeft: {
-    position: 'absolute',
-    bottom: 0,
-  },
-  fixedSectionRight: {
-    position: 'absolute',
-    bottom: 10,
-    right: 10,
-  },
-});
+const Background = styled.View`
+  position: absolute;
+  top: 0;
+  background-color: ${colors.primaryDark};
+  height: ${props => props.height};
+`;
+
+const StickySection = styled.View`
+  height: ${STICKY_HEADER_HEIGHT};
+  background-color: ${colors.primaryDark};
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const StickySectionText = styled.Text`
+  color: ${colors.white};
+  font-family: ${styledFonts.fontPrimaryBold};
```

Comment:
instead of:
`font-family: ${styledFonts.fontPrimaryBold};`

use: 
`${fonts.fontPrimaryBold};` (without specifying font-family)